routetable/route_table.go

```diff
@@ -122,6 +122,11 @@ func (r *RouteTable) SetRoutes(ifaceName string, targets []Target) {
 	r.dirtyIfaces.Add(ifaceName)
 }
 
+func (r *RouteTable) QueueResync() {
+	r.logCxt.Info("Queueing a resync.")
+	r.inSync = false
```

Comment:
30s seems a bit long, but I guess acceptable considering that we're dealing with Enemy Action :)